batch deploys fail

```
gsutil cat gs://hail-ci-0-1/deploy/ef349a51016f\*/job-log
```

the last few lines:

```
+ make push-batch
docker build -t batch .
time="2018-09-26T00:14:20Z" level=error msg="failed to dial gRPC: cannot connect to the Docker daemon. Is 'docker daemon' running on this host?: dial unix /var/run/docker.sock: connect: permission denied"
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post http://%2Fvar%2Frun%2Fdocker.sock/v1.38/build?buildargs=%7B%7D&cachefrom=%5B%5D&cgroupparent=&cpuperiod=0&cpuquota=0&cpusetcpus=&cpusetmems=&cpushares=0&dockerfile=Dockerfile&labels=%7B%7D&memory=0&memswap=0&networkmode=default&rm=1&session=vhnl6wchhs00sgt8raa35j7m7&shmsize=0&t=batch&target=&ulimits=null&version=1: dial unix /var/run/docker.sock: connect: permission denied
time="2018-09-26T00:14:20Z" level=error msg="Can't add file /hail/repo/batch/batch/server.py to tar: io: read/write on closed pipe"
Makefile:14: recipe for target 'build-batch' failed
make: *** [build-batch] Error 1
```

this is failing all deploys of hail, which is safe, but it prevents our users from getting updates.

still failing:
```gsutil cat gs://hail-ci-0-1/deploy/c28a3f9863a0\*/job-log ```
```
+ gcloud -q auth activate-service-account --key-file=/secrets/gcr-push-service-account-key.json
Activated service account credentials for: [gcr-push@broad-ctsa.iam.gserviceaccount.com]
+ gcloud -q auth configure-docker
Docker configuration file updated.
+ make push-batch
docker build -t batch .
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post http://%2Fvar%2Frun%2Fdocker.sock/v1.38/build?buildargs=%7B%7D&cachefrom=%5B%5D&cgroupparent=&cpuperiod=0&cpuquota=0&cpusetcpus=&cpusetmems=&cpushares=0&dockerfile=Dockerfile&labels=%7B%7D&memory=0&memswap=0&networkmode=default&rm=1&session=nt5ube8nzit2kbdia2afrfify&shmsize=0&t=batch&target=&ulimits=null&version=1: dial unix /var/run/docker.sock: connect: permission denied
make: *** [build-batch] Error 1
Makefile:14: recipe for target 'build-batch' failed

```

--- a/ci/deployment.yaml
+++ b/ci/deployment.yaml
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
         - name: hail-ci
-          image: gcr.io/broad-ctsa/hail-ci:0d5db5aa44cb934a08ce06c789cf389bd10d5d6b57a7239d60606e95b7fbb71b
+          image: gcr.io/broad-ctsa/hail-ci:102b3c13a57d1eefd5f91558b72b8cabc9d01cef89aa6f38a5bcd7fba834672c
           env:
             - name: SELF_HOSTNAME
               value: http://hail-ci

